pkg/util/timeutil/time_zone_util.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 The Cockroach Authors.
+// Copyright 2019 The Cockroach Authors.
```

Comment:
nit: we don't have to change this :)

Reply:
I read that part of the wiki after this commit. :)